support-frontend/assets/pages/weekly-subscription-checkout/components/weeklyCheckoutForm.tsx

```diff
@@ -162,6 +174,8 @@ function WeeklyCheckoutForm(props: PropTypes) {
 			props.price,
 			props.billingPeriod,
 		);
+		inThreeTierTestVariant &&
+			props.setPaymentMethod({ paymentMethod: 'Stripe' });
```

Comment:
What's this doing?

Reply:
This is there in order to preselect the stripe (card) radio option in the payment options form, as recommended with Helene ... It's not the ideal place to put this in the useeffect hook but it is the simplest and as the payment option form is likely to be below the fold we haven't noticed any page shift with this approach.